pkg/asset/installconfig/platformprovisioncheck.go

```diff
@@ -64,7 +65,12 @@ func (a *PlatformProvisionCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return err
 		}
-	case aws.Name, baremetal.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name:
+	case baremetal.Name:
```

Comment:
Could we fix the alphabetization of this switch? vsphere should go to the bottom, and baremetal after azure. Let's leave this better than we found it :)